buzzmobile/launch/simulation.launch

```diff
@@ -9,6 +9,7 @@
         <include file="$(find buzzmobile)/launch/includes/mapping.launch"/>
         <include file="$(find buzzmobile)/launch/includes/steering.launch"/>
         <include file="$(find buzzmobile)/launch/includes/lidar.launch"/>
+        <node pkg="buzzmobile" name="car_pose_mux" type="car_pose_mux.py"/>
```

Comment:
I don't know how much I like this solution.. I'd rather have a redundant controller node started (and not doing anything) than launch nodes directly. Others can pitch in but in either case this change doesn't belong in this PR